packages/dm-core-plugins/src/list/ListPlugin.tsx

```diff
@@ -108,7 +108,12 @@ export const ListPlugin = (props: IUIPlugin & { config?: TListConfig }) => {
       return
     }
 
-    const view = { ...internalConfig.openViewConfig, label: item?.data?.name }
+    const view = {
+      ...internalConfig.openViewConfig,
+      label: item?.data?.name
+        ? item?.data?.name
+        : `${attribute?.name} #${item.index}`,
```

Comment:
I think this can be simplified to this?
```suggestion
      label: item?.data?.name ?? `${attribute?.name} #${item.index}`,
```

Reply:
Agreed! 👍